pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -65,6 +65,9 @@ type NestedPendingOperations interface {
 	// IsOperationPending returns true if an operation for the given volumeName and podName is pending,
 	// otherwise it returns false
 	IsOperationPending(volumeName v1.UniqueVolumeName, podName types.UniquePodName) bool
+
+	// GetOperations returns the list of operations
+	GetOperations() []operation
```

Comment:
Operations are an implementation detail that should not be exposed directly in the public API. See if you could boil down exactly what the tests that need this require, and if you can use an existing API for that. I guess the problem is that you are doing white box testing to verify an implementation detail (operations running in parallel or not). But I think you should be able to deduce that without having to add a new method.